airtime_mvc/application/views/scripts/index/index.phtml

```diff
@@ -43,261 +23,245 @@
         <p><?php echo $this->escape($this->stationDescription) ?></p>
     </div>
 
-    <div id="tab-3" class="login tab_content">
-        <h2>Login</h2>
-        <form action="demo_form.asp">
-            <input type="text" name="FirstName" value="username"><br>
-            <input type="text" name="Password" value="password"><br>
-            <input type="submit" value="Login">
-        </form>
-    </div>
-
-    <?php
-    if ($this->displayRssTab) {?>
-        <div id="tab-4" class="podcast tab_content">
-            <div class="rss-title-wrapper">
-                <div id="rss-subscribe">
-                    <a target="_blank" href="<?php echo $this->stationPodcastRssUrl ?>" class="rss_icon"><span><?php echo _("Subscribe") ?></span><img src="css/radio-page/img/rss.png"></a>
-                </div>
-                <div id="rss-title">
-                    <?php echo $this->podcastTitle ?>
-                </div>
+    <div id="tab-4" class="podcast tab_content">
+        <div class="rss-title-wrapper">
```

Comment:
Not checking `displayRssTab ` seems to break when stations do not have a public podcast (see above for details).

Reply:
I meant to only remove the extra login form 🤦